src/main/java/net/nullspace_mc/tapestry/loggers/TPSLogger.java

```diff
@@ -14,7 +14,7 @@ public String getName() {
     }
 
     public Text tickLogger(MinecraftServer server, ServerPlayerEntity player) {
-        double mspt = MathUtil.round(2, MathHelper.average(server.serverTickTimes) * 1.0E-6D);
+        double mspt = MathUtil.round(2, MathHelper.average(server.averageSentPacketCount) * 1.0E-6D);
```

Comment:
yes I know this is sussy as a wussy, it's an error in the mappings, it'll be fixed in a future build